### ISSUE TITLE
Text Element does not open media manager

### ISSUE THREAD

**Issue by nidzo80** [templaza/astroid-framework#621]
Framework Version: 3.0.12
Template Two Version: 1.0.3
Joomla 5.0.3

I noticed that text element does not open JCE file manager if I want to insert image or video. The only way is to insert url of image or video wich is inconvenient way.

![2024-03-28_202406](https://github.com/templaza/astroid-framework/assets/21330927/53414241-d75c-41fd-bf1e-d7a85c086deb)

**Comment by Chacapamac** [comment]
Yes, if you primary Editor is JCE, it will maybe preferable if JCE is Use in Astroid Template manager.

**Comment by nidzo80** [comment]
> Yes, if you primary Editor is JCE, it will maybe preferable if JCE is Use in Astroid Template manager.

I don't understand you. It is same when I switch to Tiny MCE.

**Comment by sonvnn** [comment]
@nidzo80 Astroid is using its editor. For some reason I can not integrate Joomla editor yet.

**Comment by nidzo80** [comment]
@sonvnn it wa my mistake. It was opening directly Windows explorer. This is from 
Framework Version: 3.0.11
Template Version: 1.0.3


https://github.com/templaza/astroid-framework/assets/21330927/3b3a18da-140c-4ef3-a643-c9aa0e681e05

**Comment by Chacapamac** [comment]
@nidzo80  Can I mark this Post as “RESOLVED”?

**Comment by nidzo80** [comment]
@Chacapamac  No. Framework Version: 3.0.12 should open Windows explorer too like 3.0.11 does.

**Comment by Chacapamac** [comment]
What element or position you are editing here?

**Comment by felixkat** [comment]
> [@nidzo80](https://github.com/nidzo80) Astroid is using its editor. For some reason I can not integrate Joomla editor yet.

To prevent confusion, I am closing this issue as TinyMCE has now replaced the Astroid editor.
